views/vibelog.post.dt

```diff
@@ -24,27 +24,9 @@ block vibelog-content
 					p Posted at #{toRFC822TimeString(info.post.date)} by #{info.post.author in info.users ? info.users[info.post.author].name : info.post.author}
 
 			section.comments
-				- if( info.comments.length )
-					h2#comments Comments
-					
-					- import vibe.textfilter.html;
-					- foreach( c; info.comments )
-						.comment
-							.commentHeader
-								- if( c.authorName.length > 0 && c.authorHomepage.length > 0 )
-									a(href="#{c.authorHomepage}")= c.authorName
-								- else if( c.authorName.length > 0 )
-									| #{c.authorName}
-								- else
-									| anonymous
-								- if( c.authorMail.length > 0 )
-									|!= "&lt;" ~ htmlAllEscape(c.authorMail) ~ "&gt;"
-								| on #{toRFC822DateString(c.date)}, #{toRFC822TimeString(c.date)}:
-							.commentContent= c.content
-				
 				- if( info.post.commentsAllowed )
 					h2 Post a comment
-					form(action="#{info.post.name}/post_comment", method="POST")
+					form(action="#{info.refPath}/post_comment", method="POST")
```

Comment:
This would fail if the web interface is registered in a sub path (e.g. http://vibed.org/blog/).